src/OrbitGl/Track.cpp

```diff
@@ -143,9 +142,8 @@ void Track::Draw(Batcher& batcher, TextRenderer& text_renderer, uint64_t current
     Vec2 tab_top_right(top_left[0] + label_width, top_left[1]);
     Vec2 tab_bottom_right(top_left[0] + label_width, top_left[1] - label_height + top_margin);
     Vec2 content_bottom_left(top_left[0] - tab_x0, top_left[1] - size_[1]);
-    Vec2 content_bottom_right(top_left[0] + size_[0] - right_margin, top_left[1] - size_[1]);
-    Vec2 content_top_right(top_left[0] + size_[0] - right_margin,
-                           top_left[1] - label_height + top_margin);
+    Vec2 content_bottom_right(top_left[0] + size_[0], top_left[1] - size_[1]);
+    Vec2 content_top_right(top_left[0] + size_[0], top_left[1] - label_height + top_margin);
```

Comment:
We could also erase top_margin, what do you think?

Reply:
`top_margin` is the space between the track top margin and the track content top. What do you mean by "erase" it?

Reply:
Doing the same thing we did with the right_margin and including it when deciding the rectangle where we want to draw

Reply:
I don't think so. The top margin is a part of the track. It is a space between the track top and the content top https://screen/AdEUTT8iGriZUev.

`top_margin` will not decide the location of the track, it will affect the space arrangement inside a track. Why to do the same thing like `right_margin`?